plugins/modules/update_systems.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2022, Chris Santiago (chsantia@redhat.com)
+# MIT License (see LICENSE or https://opensource.org/licenses/MIT)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: update_systems
+short_description: Update systems
+description:
+  - This module will update systems to the latest update or to a specific version
+version_added: "0.1.0"
+options:
+  systems:
+    description:
+      - Systems that are passed from the filter systems module to be updated.
```

Comment:
Let's go ahead and reference this by Fully Qualified Collection Name (FQCN) for good measure. This would be `consoledot.edgemanagement.filter_systems` in this example. I would also suggest we wrap that in `C()` for docs parsing.

Ultimately, I would recommend changing it to:

```
- Systems that are passed from the C(consoledote.edgemanagement.filter_systems) module to be updated.
```